### DISCUSSION TITLE
github package registry authentication of organisation under maven

### DISCUSSION THREAD

**Discussion by tombrus** [community/community#26469] [Actions]
<p>I constructed a <strong>settings.xml</strong> file for uploading files in the  <strong>package registry</strong> through maven. For the <strong>USERNAME</strong> I entered <strong>my personal email address</strong> and for the password I used <strong>${{ </strong>** secrets.GITHUB_TOKEN }}**. This works fine.</p>
<p>But I have not found a way to authenticate as my organisation. I tried the name, the display name and the email, nothing works (<em>Access denied</em>). I do not like to put my personal email address in the workflow file of my organistation.</p>
<p>Is there a solution to authenticate as the organisation?</p>

**Comment by tombrus** [comment, marked as answer]
<p>oops, sorry, typo, you can use the name of the organisation.</p>

**Comment by artieren** [comment]
<p>Hello,</p>
<p>I experienced same issue with my organization. I tried many things according this page <a href="https://help.github.com/en/packages/using-github-packages-with-your-projects-ecosystem/configuring-apache-maven-for-use-with-github-packages#installing-a-package" rel="nofollow noopener">https://help.github.com/en/packages/using-github-packages-with-your-projects-ecosystem/configuring-apache-maven-for-use-with-github-packages#installing-a-package</a> but nothing happen.  <strong>mvn deploy</strong> command couse this error: <strong>Failed to retrieve remote metadata com.artimetrik.isg:health-surveillance:0.0.1-SNAPSHOT/maven-metadata.xml: Could not transfer metadata com.artimetrik.isg:health-surveillance:0.0.1-SNAPSHOT/maven-metadata.xml from/to github-nexus (<a href="https://maven.pkg.github.com/artimetrik/health-surveillance" rel="nofollow noopener">https://maven.pkg.github.com/artimetrik/health-surveillance</a>): Not authorized</strong></p>
<p>Could you help me please?</p>
